install_on_olinuxino.md

```diff
@@ -0,0 +1,106 @@
+# Installation sur une carte OlinuXino
+
+<div class="alert alert-info" markdown="1">
+Pour installer YunoHost sur une carte OLinuXino, le plus simple est d'utiliser l'image du projet [Site du projet *La Brique Internet*](http://labriqueinter.net). Il s'agit d'une image YunoHost qui est actuellement spécifiquement destinée aux cartes OLinuXino.
+</div>
+
+<div class="alert alert-warning" markdown="1">
+Si vous souhaitez mettre en place ou obtenir une Brique Internet complète (Carte Olimex + YunoHost + VPN associatif neutre + Hotspot Wifi), il est dans ce cas préférable d'utiliser [la procédure d'installation du projet La Brique Internet](https://install.labriqueinter.net).
+
+Pour faire votre choix, nous vous conseillons de rencontrer votre FAI associatif local et de consulter [les avantages d'un VPN neutre dans le cadre de l'auto-hébergement](/vpn_advantage_fr).
+</div>
+
+## Prérequis
+
+Le matériel nécessaire :
+* Un mini-serveur Olimex :
+ * [A20-OLinuXino-LIME](https://www.olimex.com/Products/OLinuXino/A20/A20-OLinuXino-LIME/open-source-hardware)
+ * [A20-OLinuXino-LIME2](https://www.olimex.com/Products/OLinuXino/A20/A20-OLinuXino-LIME2/open-source-hardware)
+* Une carte micro-SD (des [Transcend 300x](http://www.amazon.fr/Transcend-microSDHC-adaptateur-TS32GUSDU1E-Emballage/dp/B00CES44EO) pour des raisons de performance/stabilité).
+* Un adaptateur secteur [européen](https://www.olimex.com/Products/Power/SY0605E/) pour alimenter la carte olimex. L’alimentation via USB semble peu stable.
+* Un câble Ethernet/RJ-45 pour brancher la Brique à son routeur.
+
+Et évidemment, **un ordinateur sous GNU/Linux ou BSD**.
```

Comment:
This note was already there in previous version, but it's not quite friendly for people running Windows :/ ... Flashing an image on an SD card on windows is already documented [here](https://yunohost.org/#/copy_image) so we could point Windows user to it. Don't know about arp-scan but I think the command exist on Windows as well (even though Windows's CLI is crap). Then we'd need to tell people to use PuTTy (or some other client if anybody knows one).

But we can add this later if nobody feels like documenting it.